media/css/firefox/whatsnew/whatsnew-95-de.scss

```diff
@@ -0,0 +1,169 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-pocket';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/components/picto';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+@import 'includes/header';
+
+// * -------------------------------------------------------------------------- */
+// Main content
+
+.wnp-content-main {
+    padding: 0;
+    text-align: center;
+}
+
+.wnp-main-title {
+    @include text-title-md;
+    color: get-theme('title-text-color');
+}
+
+.wnp-main-tagline {
+    @include text-body-lg;
+}
+
+.wnp-main-cta {
+    margin-top: $layout-md;
+}
+
+.mzp-c-logo {
+    margin: 0 auto $spacing-xl;
+}
+
+// * -------------------------------------------------------------------------- */
+// Extra content
+
+.mzp-c-picto {
+    text-align: center;
+    max-width: $content-sm;
+
+    @media #{$mq-lg} {
+        @include bidi(((text-align, left, right),));
+    }
+}
+
+.mzp-c-picto-image svg {
+    height: auto;
+    max-width: 100%;
+}
+
+// * -------------------------------------------------------------------------- */
+// Utilities (link to release notes)
+
+.c-utilities {
+    @include text-body-sm;
+    max-width: $content-md;
+    padding-bottom: $layout-xl;
+    text-align: center;
+}
+
+// * -------------------------------------------------------------------------- */
+// Footer overrides
```

Comment:
was there a request to change the footer? it seems to be the regular grey for both light and dark themes in the mockups

for light theme, the link styling is a bit off:
- regular links don't get an underline on hover like they do on the dark theme
<img width="157" alt="Screen Shot 2021-11-26 at 11 25 40 AM" src="https://user-images.githubusercontent.com/19650432/143574161-3b67a64b-3c61-4bd3-b6c3-6e10d958f3ce.png">

- the social icons do get an underline on hover, but I don't think that's the desired look
<img width="164" alt="Screen Shot 2021-11-26 at 11 25 05 AM" src="https://user-images.githubusercontent.com/19650432/143574190-2553c5ad-a7af-4c2f-8ca4-6687cd3ae69a.png">

Reply:
@maureenlholland this is the standard footer theme we use for WNPs, no new changes added here just reuse of existing code from prior WNPs.

Reply:
I will see if we can improve the icon hover styling though 👍